ChangeLog

```diff
@@ -1,3 +1,14 @@
+2024-11-03  Marco Colombo <mar.colombo13@gmail.com>
```

Comment:
Two spaces between date and name (which you have) and name and email (one missing).

One empty line after the header.

Reply:
Done :smile: